gapis/api/sync/sync.go

```diff
@@ -42,7 +42,9 @@ type SynchronizedAPI interface {
 	ResolveSynchronization(ctx context.Context, d *Data, c *path.Capture) error
 
 	// MutateSubcommands mutates the given Atom calling callback after each subcommand is executed.
-	MutateSubcommands(ctx context.Context, id api.CmdID, cmd api.Cmd, s *api.State, callback func(*api.State, api.SubCmdIdx, api.Cmd)) error
+	MutateSubcommands(ctx context.Context, id api.CmdID, cmd api.Cmd, s *api.State,
```

Comment:
Please update godoc for `preSubCmdCallback` and `postSubCmdCallback`. I'm completely unsure what they mean.